### PR TITLE
Removed always-false tests + other minor refact

### PR DESCRIPTION
**Pull request by xylographe**

### CODE REVIEW

File: src/Forms/SetMinimumDisplayTimeBetweenParagraphs.cs [hunk 1]
```diff
@@ -72,8 +70,10 @@ public Subtitle FixedSubtitle
         public void Initialize(Subtitle subtitle)
         {
             _subtitle = subtitle;
-            numericUpDownMinMillisecondsBetweenLines.Value = Configuration.Settings.General.MininumMillisecondsBetweenLines;
-            //            GeneratePreview();
+            numericUpDownMinMillisecondsBetweenLines.Value = Configuration.Settings.General.MinimumMillisecondsBetweenLines != 0
+                                                           ? Configuration.Settings.General.MinimumMillisecondsBetweenLines
+                                                           : 1;
+            //GeneratePreview();
```
Comment (xylographe) [inline src/Forms/SetMinimumDisplayTimeBetweenParagraphs.cs]: Did throw ArgumentOutOfRangeException when `MinimumMillisecondsBetweenLines == 0`

File: src/Logic/Settings.cs [hunk 2]
```diff
@@ -1316,9 +1316,11 @@ private static Settings CustomDeserialize(string fileName)
             subNode = node.SelectSingleNode("SubtitleMaximumDisplayMilliseconds");
             if (subNode != null)
                 settings.General.SubtitleMaximumDisplayMilliseconds = Convert.ToInt32(subNode.InnerText);
-            subNode = node.SelectSingleNode("MininumMillisecondsBetweenLines");
+            subNode = node.SelectSingleNode("MinimumMillisecondsBetweenLines");
+            if (subNode == null) // TODO: Remove in 3.5
+                subNode = node.SelectSingleNode("MininumMillisecondsBetweenLines");
```
Comment (xylographe) [inline src/Logic/Settings.cs]: Backwards compatible

File: src/Logic/Settings.cs [hunk 3]
```diff
@@ -2051,9 +2053,11 @@ private static Settings CustomDeserialize(string fileName)
             subNode = node.SelectSingleNode("SpectrogramAppearance");
             if (subNode != null)
                 settings.VideoControls.SpectrogramAppearance = subNode.InnerText;
-            subNode = node.SelectSingleNode("WaveformMininumSampleRate");
+            subNode = node.SelectSingleNode("WaveformMinimumSampleRate");
+            if (subNode == null) // TODO: Remove in 3.5
+                subNode = node.SelectSingleNode("WaveformMininumSampleRate");
```
Comment (xylographe) [inline src/Logic/Settings.cs]: Backwards compatible

File: src/Logic/Settings.cs (hunk 2 above)
Comment (niksedk) [inline src/Logic/Settings.cs]: Probably better just to fix the language files...

Comment (xylographe) [inline src/Logic/Settings.cs]: ???
I'm afraid, I don't understand what you mean.
These two 'compatibility lines' are there to be able to use the value from the existing Settings.xml file (with the tag named Mini**n**umMillisecondsBetweenLines).
When the Settings.xml file is re-generated, the new tag (Mini**m**umMillisecondsBetweenLines) will be used.

Comment (niksedk) [inline src/Logic/Settings.cs]: ups - I thought it was language tags...

**Pull request merged**